client/src/components/DragAndDrop/reorder.jsx

```diff
@@ -0,0 +1,35 @@
+
+// REORDERING THE RESULT
+const reorder = (list, startIndex, endIndex) => {
+    const result = Array.from(list);
+    const [removed] = result.splice(startIndex, 1);
+    result.splice(endIndex, 0, removed);
+    return result;
+};
+export default reorder;
```

Comment:
Let's move this to the end of the file. I would also recommend using a named import here (e.g. `export const reorder ...` as we've used that syntax below).